doc/development/create-a-model.md

```diff
@@ -22,7 +25,119 @@ class SomeDescript(Descriptor):
     def __init__(self, arg1: bool, arg2: float) -> None:
         pass
 ```
+:::
+
+:::{tab-item} PyTorch {{ pytorch_icon }}
+The PyTorch backend follows a meticulously structured inheritance pattern. When creating a new descriptor, it is essential to inherit from both the {py:class}`deepmd.pt.model.descriptor.base_descriptor.BaseDescriptor` class and the {py:class}`torch.nn.Module` class. Abstract methods, including {py:class}`deepmd.pt.model.descriptor.base_descriptor.BaseDescriptor.fwd`, must be implemented, while others remain optional. It is crucial to adhere to the original method arguments without any modifications. Once the implementation is complete, the next step involves registering the component with a designated key:
```

Comment:
```suggestion
The PyTorch backend follows a meticulously structured inheritance pattern. When creating a new descriptor, it is essential to inherit from both the {py:class}`deepmd.pt.model.descriptor.base_descriptor.BaseDescriptor` class and the {py:class}`torch.nn.Module` class. Abstract methods, including {py:class}`deepmd.pt.model.descriptor.base_descriptor.BaseDescriptor.forward`, must be implemented, while others remain optional. It is crucial to adhere to the original method arguments without any modifications. Once the implementation is complete, the next step involves registering the component with a designated key:
```

Reply:
please check https://github.com/deepmodeling/deepmd-kit/blob/a7d44d1c2283bcc4a096b1f0eb5eaf6241078ef1/deepmd/pt/model/descriptor/base_descriptor.py#L8 that rename the name of the forward method.

Reply:
many abstract methods needs to be implemented. please check deepmd/dpmodel/descriptor/make_base_descriptor.py